daq/topology.py

```diff
@@ -428,17 +428,17 @@ def _add_dot1x_incoming_rule(self, incoming_acl, secondary_acl):
                     vlan = self._get_port_vlan(device.port.port_no)
                     test_ports = device.gateway.get_possible_test_ports()
                     if test_ports:
-                        self._add_dot1x_reflector(incoming_acl, vlan, test_ports, True)
+                        self._add_dot1x_allow_rule(incoming_acl, test_ports, vlan_vid=vlan)
                     device_port = device.port.port_no
                     if device_port:
-                        self._add_dot1x_reflector(secondary_acl, vlan, [device_port], True)
+                        self._add_dot1x_allow_rule(secondary_acl, [device_port], vlan_vid=vlan)
 
-    def _add_dot1x_reflector(self, acl, vlan, ports, is_trunk):
+    def _add_dot1x_allow_rule(self, acl, ports, vlan_vid=None, out_vlan=None):
         """Add dot1x reflection rule to acl"""
-        if is_trunk:
-            self._add_acl_rule(acl, vlan_vid=vlan, eth_type=self._DOT1X_ETH_TYPE, ports=ports)
-        else:
-            self._add_acl_rule(acl, eth_type=self._DOT1X_ETH_TYPE, ports=ports, out_vlan=vlan)
+        if vlan_vid:
+            self._add_acl_rule(acl, vlan_vid=vlan_vid, eth_type=self._DOT1X_ETH_TYPE, ports=ports)
```

Comment:
keep order of arguments consistent as much as possible with the line 2 below this.   acl, ports, eth_type, *others*